crates/ef-testing/src/models/case.rs

```diff
@@ -165,10 +169,15 @@ impl BlockchainTestCase {
                 actual -= transaction_cost;
             }
             if actual != expected_state.balance.0 {
-                return Err(RunnerError::Other(format!(
+                let balance_diff = format!(
                     "balance mismatch for {:#20x}: expected {:#32x}, got {:#32x}",
                     address, expected_state.balance.0, actual
-                )));
+                );
+                diff.push(balance_diff);
+            }
+
+            if !diff.is_empty() {
+                return Err(RunnerError::Other(diff.join("\\n")));
```

Comment:
I think we can remove this if it doesn't work

Reply:
but I still need to input a string to `Other` and `diff` is a `Vec<String>`

Reply:
Will be fixed in another PR